contracts/MerkleDelegation.sol

```diff
@@ -76,17 +61,24 @@ contract MerkleDelegation is Ownable, Pausable {
         view
         returns (bytes32 trieRoot)
     {
-        require(delegator != address(0), "DR: delegator must be non-zero");
-        return delegation[delegator].trieRoot;
+        require(delegator != address(0), "delegator must be non-zero");
+        if (delegation[delegator].trieRoot != bytes32(0)) {
+            return delegation[delegator].trieRoot;
+        } else {
+            return
+                bytes32(
+                    0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421
```

Comment:
Does it make sense to return this value? I was under the impression that by setting the trie root to that of an empty merkle trie was to differentiate between an uninstanciated delegate (having the value of `bytes32(0)`) and a previously instanciated delegate that has been cleared (having the value `0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421`). 

If we are not differentiating i would argue that we should be clearing the slot in the function `clearDelegateTrie` for the gas refund rather than writing a different value to it.

Reply:
Thats fair, I was really just wanting to make it clear to anyone checking the contract that we're pointing to an empty trie, but the caller can make the same assumption with `bytes32(0)`. 

Our api doesn't read from the contract state, its just reading event logs, so we don't need these return values at all. I will set it back to `bytes32(0)` 👍

Reply:
Sounds good, I would keep this getter regardless as it would be nice for users to be able to query directly from the contract for verifiability

Reply:
Yes!! Addressed in [ad46fca](https://github.com/windranger-io/windranger-mass-delegation-contracts/pull/17/commits/ad46fcad2d4126ea31e7c07a32937a5f2a84142f) 🙌